src/components/behandlerdialog/meldingtilbehandler/VelgBehandler.tsx

```diff
@@ -111,7 +114,15 @@ export const VelgBehandler = ({
           behandlerRefField.onChange(event);
         }}
       >
-        {texts.behandlersokTekst}
+        <div className="flex flex-row">
+          {texts.behandlersokTekst}
+          <HelpText
+            title={texts.sokEtterBehandlerHelpTextTitle}
+            className="ml-1"
+          >
+            {texts.sokEtterBehandlerHelpText}
+          </HelpText>
```

Comment:
Blir det rart at det er to klikkbare elementer her? Både HelpText og selve radioknappen. Ser at man får hover-effekten til radioknappen på HelpText-knappen 🤔

Reply:
Synes vi bare skal gå for det foreløpig. Har oppdatert screen recording i PR beskrivelsen 😊 Bra catch! 🎣